### ISSUE TITLE
Saving a netedit config loaded in netconvert format  leads to saving the network in two formats

### ISSUE THREAD

**Issue by palvarezlopez** [eclipse-sumo/sumo#15202]
<input>
        <additional-files value="additionals.add.xml"/>
        <route-files value="routes.rou.xml"/>
        <data-files value="datas.dat.xml"/>
        <sumo-net-file value="netedit.net.xml"/> 
        <node-files value="netccfg.nod.xml"/>
        <edge-files value="netccfg.edg.xml"/>
        <connection-files value="netccfg.con.xml"/>
        <tllogic-files value="netccfg.tll.xml"/>
    </input>

**Comment by palvarezlopez** [comment]
@namdre if we load a netedit config with netconvert format (node-files, edges-files, etc.)... Should we save the netedit config also in netconvert format?

**Comment by namdre** [comment]
No. The plain-xml files only serve as a patch on the loaded net file and it doesn't make sense to save it as plain-xml in response. Just save it as .net.xml

**Comment by behrisch** [comment]
plain XML should only be written if requested explicitly
